Assets/Scripts/Game/UI/DamageNumber.cs

```diff
@@ -0,0 +1,43 @@
+using UnityEngine.UI;
+using UnityEngine;
+using System.Collections;
+using TMPro;
+using Queue.Systems.UISystem;
+using DG.Tweening;
+
+
+public class DamageNumber : MonoBehaviour
+{
+    [SerializeField] TextMeshPro damageNumber;
+    [SerializeField] RectTransform rectTransform;
+    [SerializeField] RectTransform canvasRectTransform;
+    [SerializeField] float apearanceTime;
+    public bool isAvailable = true;
+
+    void Start()
+    {
+        isAvailable = true;
+    }
+
+    public void ActivateDamageNumber(float num, Vector3 pos)
+    {
+        Debug.Log($"Displaying damage number {num}");
+        isAvailable = false;
+        damageNumber.text = num.ToString();
+        pos = new Vector3(pos.x, pos.y + 5, pos.z);
+        rectTransform.position = pos;
+        StartCoroutine(DisplayDamageNumbers());
+
+    }
+
+    private IEnumerator DisplayDamageNumbers()
+    {
```

Comment:
Hardcoded values. Exposed those values via a serialized field. You can use Odin Inspector's box group or tab group properties in order to keep the editor organized when you add those values.

https://odininspector.com/attributes/box-group-attribute